python/aida_interchange/aifutils.py

```diff
@@ -613,10 +647,23 @@ def add_source_document_to_justification(g, justification, source_document) :
     :returns: The modified justification
     :rtype: rdflib.term.BNode
     """
-    g.add((justification, AIDA_ANNOTATION.sourceDocument,
+    g.add((justification, interchange_ontology.sourceDocument,
             Literal(source_document, datatype=XSD.string)))
     return justification
 
+def mark_handle(g, to_mark, handle) -> URIRef:
+    """
```

Comment:
I only see mark_handle being used once. Did you pull this code out because you anticipate using it elsewhere?

Reply:
In a sense. This is intended to be a public API, which is why it's made into its own function. See AIFUtils.java:1151